build.cake

```diff
@@ -13,9 +13,9 @@ using System.Net;
 using System.Threading;
 
 // Native SDK versions
-const string AndroidSdkVersion = "3.0.1-1+f4bafd4b2";
-const string IosSdkVersion = "3.0.1-9+b0b0b83295631713675ce8c40147f2ca6115c60f";
-const string UwpSdkVersion = "3.0.1-r0010-44b04cb";
+const string AndroidSdkVersion = "3.1.0";
+const string IosSdkVersion = "3.1.0";
+const string UwpSdkVersion = "3.1.0";
```

Comment:
We must update cgmanifest for this versions

Reply:
I'm not sure how to update cgmanifest.

Reply:
I've added this point to release instruction